modules/simple-payments/simple-payments.php

```diff
@@ -244,9 +244,9 @@ function output_shortcode( $data ) {
 			$image,
 			esc_attr( "${css_prefix}-details" ),
 			esc_attr( "${css_prefix}-title" ),
-			$data['title'],
+			wp_kses( $data['title'] ),
```

Comment:
Seeing a visible error on Jurassic
```
Warning: Missing argument 2 for wp_kses(),
```

https://developer.wordpress.org/reference/functions/wp_kses/

I'd say we can also revisit if we want to allow any html here or in the description. eg maybe simply `esc_html`

Reply:
Updated by @simison on https://github.com/Automattic/jetpack/commit/c539d21656f2e065ad86db0b5f9836fe7877d69e.